lib/gateways/fixed_data_gateway.rb

```diff
@@ -0,0 +1,9 @@
+class FixedDataGateway
+  def save(fixed_data)
+    FixedDatum.create(fixed_data)
```

Comment:
This could fail silently at the moment, as `#create` won't riase an error if `fixed_data` isn't valid. You can use `#create!` to have it raise an error instead. It might be worth moving the validation from the model to the use case, as this allows validation which works the same regardless of how we're persisting the data. e.g. if we switched from ActiveRecord to posting to an API.